sympy/tensor/tensor.py

```diff
@@ -54,6 +55,24 @@ def deprecate_data():
     pass
 
 
+@deprecated(useinstead="TensorSymmetry constructor and methods", issue=17108,
+            deprecated_since_version="1.5")
+def deprecate_tensorsymmetry():
```

Comment:
why not add the deprecation decorator to `tensorsymmetry`?

Reply:
Agree, same as above.